mkosi.conf

```diff
@@ -20,8 +20,9 @@ BuildSourcesEphemeral=yes
 [Host]
 @Incremental=yes
 @RuntimeSize=8G
+@QemuSmp=2
 ToolsTreePackages=virtiofsd
-KernelCommandLineExtra=systemd.crash_shell
+@KernelCommandLineExtra=systemd.crash_shell
```

Comment:
I think we can drop the `@` now that we have `--append`?

Reply:
I had intended to keep test config entirely separate so we didn't need to have command-line entries that override earlier ones, but making this mkosi config overridable instead of extendable causes problems for other config snippets.

It makes the command-line long enough that the kernel truncates when logging it which is unfortunate if logs are all you have, but is otherwise fine.